atomate/vasp/firetasks/run_calc.py

```diff
@@ -241,6 +242,60 @@ def run_task(self, fw_spec):
         runner.run(path_dir=os.getcwd())
 
 
+@explicit_serialize
+class RunAmset(FiretaskBase):
+    """
+    Run Amset directly. Requires vasprun.xml to be in current dir.
+
+    Required params:
+        material_params (dict): the static dielectric constant (epsilon_s) is
+            mandatory.
+
+    Optional params:
+        model_params (dict): parameters related to the model used and the level
+            of theory. See Amset documentation for key options.
+        performance_params (dict): parameters related to convergence, speed,
+            etc. See Amset documentation for key options
+        temperatures ([float]): the temperatures (in Kelvin)
+        doping: ([float]) doping levels you want to compute
+    """
+    required_params = ["material_params"]
+    optional_params = ["model_params", "performance_params",
+                       "temperatures", "dopings", "coeff_file",
+                       "kgrid_tp", "write_outputs", "write_inputs"]
+
+    def run_task(self, fw_spec):
+        from amset.core import Amset
+        material_params = self.get("material_params")
+        model_params = self.get("model_params", None)
+        perfs = self.get("performance_params", {})
+        perfs["max_nvalleys"] = perfs.get("max_nvalleys", 3)
```

Comment:
make sure the defaults work well for most materials. Although ideally people should fully read the Amset documentation and set these parameters intelligently, the reality is that 90% of people will just run the AmsetFW without thinking about it and report that as the "amset result". So make sure that these defaults give good numbers as they will be representative of what people report as the Amset value.